hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/ws/server/FeedCommWebSocket.java

```diff
@@ -66,19 +63,17 @@
     @OnOpen
     public void feedSessionOpen(Session session, @PathParam("feedId") String feedId) {
         MsgLogger.LOG.infoFeedSessionOpened(feedId, session.getId());
-        boolean successfullyAddedSession = connectedFeeds.addSession(feedId, session);
```

Comment:
I tend to think that having this (as we originally had it before this refactoring) a little bit different for UI clients and feeds is a good idea. I think that UI clients cannot be trusted in providing the right SessionId, while feeds must be trusted in that they know their feedIds.